resolvers/patient.js

```diff
@@ -105,7 +105,9 @@ const patientResolvers = {
             return args.id;
```

Comment:
shouldn't this be the line that gets changed? technically we should never reach line 109 as we would always return or `throw` beforehand